Codeinwp/templates-patterns-collection
# Fse template import

### Summary
- Implemented the FSE Template import inside the templates cloud

> [!NOTE]  
> For the images to be properly saved, the website needs to be public, else it will default to a placeholder image when saving the template.

### Screenshots
<details open>
    <summary>FSE Library</summary>

<img width="1420" alt="image" src="https://github.com/Codeinwp/templates-patterns-collection/assets/23024731/430055a5-bc3c-4328-81bd-5d955e0d8f4b">
</details>

<details open>
    <summary>FSE Import</summary>

<img width="1580" alt="image" src="https://github.com/Codeinwp/templates-patterns-collection/assets/23024731/20bd1687-0ff4-467d-9ab9-4042f7ba86dc">
</details>

### Test instructions
1. Use this version of the plugin
2. Use a FSE Theme
3. From the FSE Editor, save a template for the Front Page or any other page
4. Check that the template is visible inside My Library
5. Check that the Preview is visible before importing inside the Editor
6. Import the template on a different FSE Theme


Closes #255.
